docs/guides.rst

```diff
@@ -1,5 +1,8 @@
+Guides
```

Comment:
Eventually, we may have more guides and split this further apart but this might be just ok for now.